packages/openapi-to-graphql-cli/src/openapi-to-graphql.ts

```diff
@@ -125,14 +121,15 @@ Promise.all(
  * @return {object}      Content of read file
  */
 function readFile(path) {
-  try {
```

Comment:
Removed try-catch block for more streamline error handling... Not sure if this is a good practice...